core/integration/odc/plugin.go

```diff
@@ -637,6 +639,33 @@ func (p *Plugin) ObjectStack(varStack map[string]string) (stack map[string]inter
 		}
 		accumulator = append(accumulator, strings.TrimSpace(pdpGeneratorScriptPath))
 
+		odc_topology_fullname = ""
+		switch pdpConfigOption {
+		case "Repository hash":
+			odc_topology_fullname = "(hash, " + o2DPSource + ", " + pdpLibraryFile + ", " + pdpLibWorkflowName + ")"
+
+		case "Repository path":
+			odc_topology_fullname = "(path, " + o2DPSource + ", " + pdpLibraryFile + ", " + pdpLibWorkflowName + ")"
+
+		case "Manual XML":
+			odc_topology, ok := varStack["odc_topology"]
+			if !ok {
+				log.WithField("partition", envId).
+					WithField("call", "GenerateEPNWorkflowScript").
+					Error("cannot acquire ODC topology variable")
+				return
+			}
+			odc_topology_fullname = "(xml, " + odc_topology + ")"
+
+		default:
+			odc_topology_fullname = ""
```

Comment:
The whole `default` clause here is unnecessary, you already initialise it to `""`.